js/index.js

```diff
@@ -38,5 +38,96 @@ const siteContent = {
 };
 
 // Example: Update the img src for the logo
-let logo = document.getElementById("logo-img");
-logo.setAttribute('src', siteContent["nav"]["img-src"])
+const logo = document.getElementById("logo-img");
+logo.setAttribute('src', siteContent["nav"]["img-src"]);
+
+//Grab 'a' elements
+const aTags = document.getElementsByTagName('a');
+
+//Grab siteContent.nav key/value pairs
+const navKeys = Object.keys(siteContent.nav);
+const navValues = Object.values(siteContent.nav);
+
+//Set each 'a' tag to its designated key/value pair
+for (let i = 0; i < aTags.length; i++) {
+  const a = aTags[i];
+  a.setAttribute('id', navKeys[i]);
+  a.textContent = navValues[i];
+  a.style.color = 'green';
+}
```

Comment:
Nice job looping though the Nav here! Small tip: use querySelectorAll to have access to the for each method and then you could loop through it in fewer lines. for example 
```
let nav = document.querySelectorAll('a');

nav.forEach(function(value, i){
  value.textContent=siteContent.nav[`nav-item-${i+1}`];
});
```